compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -48,6 +48,36 @@ ireeCodegenQueryMMAIntrinsicsBinding(MlirOperation op) {
   return mmaList;
 }
 
+static std::optional<std::vector<int64_t>>
+ireeGPULoweringConfigAttrGetReductionTileSizesBinding(
+    MlirAttribute lowering_config) {
+  size_t tileSizes = 0;
+  ireeGPULoweringConfigAttrGetReductionTileSizes(lowering_config, &tileSizes,
+                                                 nullptr);
+  if (tileSizes == -1) {
+    return std::nullopt;
+  }
+  std::vector<int64_t> reductionTileSizes(tileSizes);
+  ireeGPULoweringConfigAttrGetReductionTileSizes(lowering_config, &tileSizes,
+                                                 reductionTileSizes.data());
+  return reductionTileSizes;
```

Comment:
We don't need the buffer to be caller-allocated -- we can return a pointer to the attribute storage.